src/Primitives.hs

```diff
@@ -676,7 +683,7 @@ primitiveDeftype _ _ _ = error "primitivedeftype"
 
 primitiveUse :: Primitive
 primitiveUse xobj ctx [XObj (Sym path _) _ _] =
-  pure $ maybe lookupInGlobal useModule (lookupInEnv path e)
+  (pure $ maybe lookupInGlobal useModule (lookupInEnv path e))
```

Comment:
The excess parenthesis will probably cause a warning in `HLint`

Reply:
4bbc6bc